lib/api/core/plugins/pluginContext.js

```diff
@@ -105,6 +105,11 @@ class PluginContext {
         }
       });
 
+      Object.defineProperty(this.accessors, 'trigger', {
+        enumerable: true,
+        get: () => curryTrigger(pluginName).bind(kuzzle)
```

Comment:
good remark, but I used the `bind` to stay consistent with the rest of the file.